### PR TITLE
feat: refactor grpo using ms-swift

### PR DESCRIPTION
**Pull request by fanqiNO1**
After merging this PR, the format of every data item should be:

```
{
    "messages": [
        {
            "role": "system",
            "content": "I am system content"
        },
        {
            "role": "user",
            "content": "I am user content"
        },
        {
            "role": "assistant",
            "content": "I am assistant content"
        }
    ],
    "images": [image_path],
    "objects": {obejctes_info}
}
```

Particularly, for fake images, `objects` should be

```
"objects": {
        "ref": [
            "I am ref1",
        ],
        "bbox": [
            [x1, x2, y1, y2]
        ]
    }
```

while for real images, `objects` is an empty dict to decide the ground truth label (`real` or `generated`) in the reward funcion.

However, for GRPO, the assistant part in the messages is optional, since it will be omitted during training.

### CODE REVIEW

**Comment by fanqiNO1** [comment]
There are some implementation difficulties in `mllm_defake.finetune.rewards.utils`, but I have no idea about how to implement them. 😿

**Review by Gennadiyev — APPROVED** [review]
